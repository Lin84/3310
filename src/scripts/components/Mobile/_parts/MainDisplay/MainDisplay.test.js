import React from 'react';
import renderer from 'react-test-renderer';

// react component:
import MainDisplay from './index';

// props:
const results = ['test', 'maindisplay'];

it('renders correctly', () => {
    const tree = renderer
        .create(<MainDisplay results={results} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
