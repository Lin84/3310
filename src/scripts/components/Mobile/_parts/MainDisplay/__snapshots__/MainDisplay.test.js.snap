// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mobile__main-display"
>
  <div
    className="mobile__result"
  >
    test
  </div>
  <div
    className="mobile__result"
  >
    maindisplay
  </div>
</div>
`;
